pkg/workers/sharings/sharing_updates.go

```diff
@@ -269,3 +269,15 @@ func ExtractHostAndScheme(fullURL string) (string, string, error) {
 	}
 	return host, scheme, nil
 }
+
+func isRecipientSide(sharing *Sharing) bool {
```

Comment:
I think it's worth telling that a recipient will never enter this part of the code if the sharing is not of type Master-Master.
The function could lead us to believe that it could return false even if it actually is called by a recipient, if the sharing type is not Master-Master for instance (which is an impossible scenario…).